js/tests/unit/collapse.js

```diff
@@ -440,4 +440,158 @@ $(function () {
       .bootstrapCollapse('show')
   })
 
+  QUnit.test('should switch between accordion and regular collapsible behavior', function (assert) {
+    assert.expect(10)
+    var done = assert.async()
+
+    var accordionHTML = '<div class="panel-group" id="accordion">'
+        + '<div class="panel"/>'
+        + '<div class="panel"/>'
+        + '<div class="panel"/>'
+        + '</div>'
+    var $groups = $(accordionHTML).appendTo('#qunit-fixture').find('.panel')
+
+    var $body1 = $('<div id="body1" aria-expanded="true" class="in"/>').appendTo($groups.eq(0))
```

Comment:
Perhaps move this HTML into `accordionHTML` and then just reference it here via `$('#body1')` ?
Ditto for body2 and body3.